WordAds: promote Ad widget within WordAds settings card

rclations opened the pull request:
The purpose of this PR is to make sure Jetpack ads module users know they can place additional ads via the Ads widget.

#### Changes proposed in this Pull Request:

* Adds help text below the `Additional ad placements` header alerting them that the Ads widget is an available option. This will only display when the Ads module is turned on - otherwise the Ads widget is not available, and users clicking the CTA would result in a poor ux.

#### Testing instructions:

* Using a Jetpack site with premium or pro plan, enable the Ads module and make sure the Ad widget message displays. A sample of what this should look like is shown below.
<img width="734" alt="screen shot 2018-04-13 at 4 11 20 pm" src="https://user-images.githubusercontent.com/2522431/38736958-5f60e1b8-3f36-11e8-8032-4f129f4c207d.png">

rickybanister commented:
Looks good to me. 

Let's monitor usage with a tracks event on the link. If it's low then we can explore something 'louder'

rclations commented:
Added tracks event on the new widget settings link. Would appreciate a review on that specifically since it's my first time working w/ tracks.

Added `Needs GDPR Review` flag in case the added tracks event needs to be accounted for anywhere.

dereksmart commented on _inc/client/traffic/ads.jsx:
```diff
@@ -31,6 +31,10 @@ export const Ads = moduleSettingsForm( class extends React.Component {
 	};
 
+	trackConfigureWidgetClick = () => {
+		analytics.tracks.recordJetpackClick( 'place-ad-widget' );
```
@withinboredom should it be a general rule that props should not use dashes? Or was #9277 a specific thing?

withinboredom replied:
anything that doesn't follow the naming conventions outlined in the tracks naming convention page on the fieldguide will be rejected. no dashes, underscores only: `<source>_<context>_<optional-subcontext>_<action>_<optional-qualifier>`

rclations replied:
proposed fixing this one in https://github.com/Automattic/jetpack/pull/9404

There are a lot of `analytics.tracks.recordJetpackClick` in jetpack that use hyphens. Are these ignored by tracks because they don't follow the convention? If so, can we switch them out without causing any other issues?
